packages/x-components/src/x-modules/tagging/service/pdp-add-to-cart.service.ts

```diff
@@ -0,0 +1,112 @@
+import { Store } from 'vuex';
```

Comment:
The methods are too big, and do too much things. Refactor them extracting some parts to make it more redable.